core/src/components/datetime/datetime.tsx

```diff
@@ -261,11 +262,36 @@ export class Datetime implements ComponentInterface {
 
     const pickerOptions = this.generatePickerOptions();
     const picker = await this.pickerCtrl.create(pickerOptions);
+
     this.isExpanded = true;
     picker.onDidDismiss().then(() => {
       this.isExpanded = false;
       this.setFocus();
     });
+    picker.addEventListener('ionPickerColChange', async (event: any) => {
```

Comment:
Did we decide on leaving this as `addEventListener`?

Reply:
Yep, spoke with Adam about this. `@Listen` won't work here since `ion-picker` isn't a child of `ion-datetime` (it's not nested in the DOM -- just dynamically appended to the dom)